README.md

```diff
@@ -14,6 +14,17 @@ Safeplaces is a toolkit for public health, built on top of data shared by users
 The project is still under development and will reach a Minimum Viable Product (MVP) stage soon.  
 *Note*: There can be breaking changes to the developing code until the MVP is released.
 
+## Accessing Uploaded Data From Clients
+
+[Safeplaces Ingest](https://github.com/Path-Check/safeplaces-backend-ingest) is a supporting backend service used by clients to upload data which is then ingested by this service. In order to interact with the upload database, the following environment variables must be set:
```

Comment:
Probably worth explicitly stating that the clients that upload data to this service include the SafePaths app.

Reply:
👍